src/tests/commentsCounter.test.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import commentCounter from '../modules/commentsCounter.js';
+
+describe('Test for comments', () => {
+  test('Should available comments', () => {
+    // Arrange
+    document.body.innerHTML = `
+       <div class="comment-section">
+         <h3 class="comment-counter"></h3>
+         <ul class="comments"></ul>
+       </div>
+     `;
+    const commentCount = commentCounter();
+
```

Comment:
To make your test more robust and efficient, I recommend tweaking the tests also to check for a case where there is no comment item in the dom.

Reply:
Hi @DuaneDave  Thank you very much for your time while reviewing our project. Thank you for the advise and insights, however I am requesting for clarity about the **test requested change** because in my testing its like have tested for the **no comment item** already.
thank you!